easybuild/easyconfigs/l/libsc/m4_files.patch

```diff
@@ -0,0 +1,13 @@
+--- Makefile.old	2019-03-05 18:37:50.912827712 +0100
```

Comment:
please include a short description of this patch above this line, as well as either author or reference to where patch comes from